lib/src/Pos/Logic/Full.hs

```diff
@@ -123,23 +117,19 @@ logicLayerFull jsonLogTx k = do
         getBlockHeader :: HeaderHash -> m (Maybe BlockHeader)
         getBlockHeader = DB.getHeader
 
+        getHashesRange
+            :: HeaderHash
+            -> HeaderHash
+            -> m (OldestFirst NE HeaderHash)
+        -- TODO CSL-2089 bug: should be Just k, but not
+        -- Nothing. Probably, listener logic is broken.
+        getHashesRange = Block.getHashesRange Nothing
```

Comment:
Should it be `Just k`? We plan to improve the horribly slow block sync by allowing a client to fetch *more* than `k` blocks (i.e. the whole chain) at once.

Reply:
In fact, it's simpler to get rid of this parameter at all. I just wanted to consult whether it is a mistake or not.